ext/capn_proto/dynamic_struct_builder.cc

```diff
@@ -17,7 +17,8 @@ namespace ruby_capn_proto {
     ClassBuilder("DynamicStructBuilder", rb_cObject).
       defineAlloc(&alloc).
       defineMethod("which", &which).
-      defineMethod("write", &write).
+      defineMethod("write", &writePacked).
```

Comment:
This seems like it should point to the write method not writePacked.

Reply:
My mistake, was cleaning up after making it work and commited this wrong.